File: aifupan_project/fupan-server/replay-common/src/main/java/com/jiuyu/replay/common/vo/SystemKvVo.java
package com.jiuyu.replay.common.vo;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serializable;
import java.util.Date;

/**
 * 系统配置的键值对信息
 *
 * @author lj
 * @email 
 * @date 2025-05-13 16:59:53
 */
@Data
@Schema(description = "系统配置的键值对信息")
public class SystemKvVo implements Serializable {
	private static final long serialVersionUID = 1L;

	/**
	 * id
	 */
	@Schema(description = "id")
	private Long id;
	/**
	 * code
	 */
	@Schema(description = "code")
	private String kvKey;
	/**
	 * value
	 */
	@Schema(description = "value")
	private String kvValue;
	/**
	 * 备注
	 */
	@Schema(description = "备注")
	private String remarks;
	/**
	 * 创建时间
	 */
	@Schema(description = "创建时间")
	private Date createDate;
	/**
	 * 最后修改时间
	 */
	@Schema(description = "最后修改时间")
	private Date updateDate;
	/**
	 * 是否已删除
	 */
	@Schema(description = "是否已删除")
	private Integer isDeleted;


}

File: aifupan_project/ReviewAnalysis/vo/system/SystemKvVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ReviewAnalysis.vo.system
{
    public class SystemKvVo
    {

        /// <summary>
        /// id
        /// </summary>
        public long? id { get; set; }

        /// <summary>
        /// code
        /// </summary>
        public string kvKey { get; set; }

        /// <summary>
        /// value 
        /// </summary>
        public string kvValue { get; set; }

        /// <summary>
        /// 备注 
        /// </summary>
        public string remarks { get; set; }

        /// <summary>
        /// 创建时间 
        /// </summary>
        public string createDate { get; set; }

        /// <summary>
        /// 最后修改时间 
        /// </summary>
        public string updateDate { get; set; }


    }
}
